installer/Run.bat

```diff
@@ -17,6 +17,8 @@
 ::
 
 @echo off
+%~d0
+cd "%~dp0"
```

Comment:
Why need quote. It worked normally without quote

Reply:
If the installation path has spaces it will be abnormal.